### PR TITLE
Fix openstack-operator for latest ironic-operator

### PR DESCRIPTION
**Pull request by rabi**
Also, fixes typo in custom-bundle.Dcokerfile

### CODE REVIEW

File: config/samples/core_v1beta1_openstackcontrolplane.yaml [hunk 1]
```diff
@@ -95,10 +95,11 @@ spec:
       ironicAPI:
         replicas: 1
         containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
-      ironicConductor:
-        replicas: 1
+      ironicConductors:
+      - replicas: 1
         containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
         pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+        storageRequest: 10G
```
Comment (gibizer) [inline config/samples/core_v1beta1_openstackcontrolplane.yaml:102]: If this passes CI (it works for me locally) then please apply the changes to the other sample https://github.com/openstack-k8s-operators/openstack-operator/blob/master/config/samples/core_v1beta1_openstackcontrolplane_collapsed_cell.yaml  as well to keep that usable.

Comment (rabi) [reply config/samples/core_v1beta1_openstackcontrolplane.yaml:102]: Ack, done.

**Review by gibizer — APPROVED** [review]
looks good. Thanks!

**Review by dprince — APPROVED** [review]

**Pull request merged**